src/index.html

```diff
@@ -1,12 +1,335 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <title>Catalog</title>
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="header__logo"><img src="./images/logo.png" alt="logo"></a>
+
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__list-item"><a href="#" class="is-active header__link">Apple</a></li>
+          <li class="header__list-item"><a href="#" class="header__link">Samsung</a></li>
+          <li class="header__list-item"><a href="#" class="header__link">Smartphones</a></li>
+          <li class="header__list-item">
+            <a href="#"
+              class="header__link"
+              data-qa="nav-hover"
+            >Laptops & Computers</a>
```

Comment:
Fix code style everywhere
```suggestion
            <a
              href="#"
              class="header__link"
              data-qa="nav-hover"
            >
              Laptops & Computers
            </a>
```